vnext/PropertySheets/Bundle.props

```diff
@@ -48,15 +48,15 @@
     <BundleEntryFile Condition="'$(BundleEntryFile)' == '' and Exists('$(BundleCommandWorkingDir)\index.windows.js')">index.windows.js</BundleEntryFile>
     <BundleEntryFile Condition="'$(BundleEntryFile)' == ''">index.js</BundleEntryFile>
 
-    <!-- Optional config file for the bundle command (metro will use metro.config.js by default) -->
-    <BundleConfigFile Condition="'$(BundleConfigFile)' == ''"></BundleConfigFile>
-
+    <FullOutDir Condition="'$([System.IO.Path]::GetFullPath($(OutDir)))' == '$(OutDir)'">$(OutDir)</FullOutDir>
```

Comment:
Should the condition use [IsPathRooted](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.ispathrooted?view=net-5.0#System_IO_Path_IsPathRooted_System_String_)? 
Technically there are many paths that are the same file but not identical, i.e. symlink expansions, long path prefix `\\?\c:`, old 8.3 paths `progra~1` (even though I haven't seen this in a long time ;)) etc, etc

Reply:
Good question for @asklar who made the original change. This is just porting it to 0.64.

Reply:
sounds like a good idea, I wasn't aware of that method when I wrote this.